modules/ROOT/pages/websockets/websocket-connector-client-components.adoc

```diff
@@ -0,0 +1,115 @@
+= About Websocket Client Components
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: connectors, ws, wss, websocket
+
+== Connector Client Components
+
+=== Open Outbound Operation
+
+This operation allows an OUTBOUND socket to establish a connection to a websocket endpoint.
+
+In order to establish a connection a configuration reference must be provided. In addition, parameters like path and uri can be provided. For further reference check connector's technical reference.
+
+The following example opens a connection reusing most of the settings defined in the configuration reference, but others like the uri are overridden.
+
+[source,xml,linenums]
+----
```

Comment:
explain what this returns... how do you get the socketId and why is that important. Show a meaningful example